src/io/output_time.cc

```diff
@@ -212,6 +256,9 @@ void OutputTime::write_time_frame()
 			// Remember the last time of writing to output stream
 			write_time = time;
 			current_step++;
+            
+            // unset validity of the output mesh after data are written at the current write time
+            is_output_mesh_valid_ = false;
```

Comment:
Possibly better to just delete the mesh  here and check its pointer for validity.